### ISSUE TITLE
Admin: Drag-and-drop for related entities

### ISSUE THREAD

**Issue by jeremyjbowers** [newsdev/nyt-entity-service#1]
Dragging an entity underneath another should create a related entity.

**Comment by jeremyjbowers** [comment]
- [x] Get a draggable JS library.
- [x] On the client, figure out when dragging the entity name and the related entity.
- [x] Make an AJAX request to the server to POST entity and related entity.
- [x] On the server, get entity and related_entity and make the relationship.
- [ ] Trap for so many different server-side errors.
- [ ] Trap for many client-side UX errors / bad behaviors.

**Comment by SHewitt95** [comment]
By chance, would the following scenario be something to trap for on the server:

User drags gray entity to pink area of another entity. User realizes error: the dragged entity actually should remain gray. User drags entity back to original gray area.

I noticed this when I did a similar action. I wrote an if-statement that checks if the entity id and the canonical id are the same. These two being the same represents the user moving a block back to its original place. If the user moves an entity back to its original place, then the canonical id of the entity is reset to "None."

Is this one trap you intended on existing?

**Comment by jeremyjbowers** [comment]
Yup. I was planning on making that an "x" button by any related entity
because the drag "syntax" is p weird for that.


contact([{ "twitter": "@jeremybowers", "phone": "727-667-0510", "github": "
http://github.com/jeremyjbowers" }])
